complex/ft_comp.c

```diff
@@ -54,13 +54,14 @@ static int ft_open_cntrs(void)
 {
 	int ret;
 
+	cntr_attr.wait_obj = test_info.wait_obj;
```

Comment:
This makes more sense.  :)  Can you match the style of ft_open_cqs()?  That is, declare the cntr_attr locally, memset it to 0, and set the values we want.  This will make it easy to modify the cntr_attr's that we want as part of the test iteration.

The wait object should probably be selected using ft_tx/rx_ctrl.comp_wait, _or_ have the cq select the wait object the same way, using the test_info, rather than control structure.  The latter might be preferable, so that we only have one field controlling this, rather than copying the comp_wait from test_info to ft_*x_ctrl.

Finally, the cntr and cq settings should be independent.